chapters/ch51.md

```diff
@@ -1,101 +1,102 @@
 
 > 1. Classifying whether an image is overexposed (like the example above)
 
---> _replace THIS LINE by your translation for the above line_
+1. Phân loại một bức ảnh có bị phơi sáng quá mức hay không (như trong ví dụ trên)
 
 > 2. Classifying whether an image was taken indoor or outdoor
 
---> _replace THIS LINE by your translation for the above line_
+2. Phân loại một bức ảnh được chụp trong nhà hay ngoài trời
 
 > 3. Classifying whether an image contains a cat
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại một bức ảnh có chứa một con mèo hay không
 
 > 4. Classifying whether an image contains a cat with both black and white fur
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại liệu một bức ảnh chứa một con mèo khoang đen trắng hay không
 
 > 5. Classifying whether an image contains a Siamese cat (a particular breed of cat)
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại liệu một bức ảnh có chứa một con mèo Siamese (tên một loại mèo)
 
 > Each of these is a binary image classification task: You have to input an image, and output either 0 or 1. But the tasks earlier in the list seem much "easier" for a neural network to learn. You will be able to learn the easier tasks with fewer training examples.
 
---> _replace THIS LINE by your translation for the above line_
+Mỗi tác vụ trong số này là một bài toán phân loại ảnh nhị phân: Bạn phải có một bức ảnh đầu vào, và cho ra hoặc 0 hoặc 1. Nhưng những tác vụ đầu tiên trong danh sách này có vẻ "dễ" hơn nhiều để một mạng neural có thể học. Bạn sẽ có thể huấn luyện những tác vụ này với ít mẫu huấn luyện hơn.
 
-> Machine learning does not yet have a good formal definition of what makes a task easy or hard.[16] With the rise of deep learning and multi-layered neural networks, we sometimes say a task is "easy" if it can be carried out with fewer computation steps (corresponding to a shallow neural network), and "hard" if it requires more computation steps (requiring a deeper neural network). But these are informal definitions.
+> Machine learning does not yet have a good formal definition of what makes a task easy or hard[16]. With the rise of deep learning and multi-layered neural networks, we sometimes say a task is "easy" if it can be carried out with fewer computation steps (corresponding to a shallow neural network), and "hard" if it requires more computation steps (requiring a deeper neural network). But these are informal definitions.
 
---> _replace THIS LINE by your translation for the above line_
+Học máy chưa có một định nghĩa chính thống nào về một tác vụ là dễ hay khó[16]. Với sự phát triển của học sâu và mạng neural đa tầng, chúng ta nói một tác vụ là "dễ" nếu có có thể được thực hiện với số lượng bước tính toán ít hơn (ứng với mạng neural nông), và "khó" nếu nó đòi hỏi nhiều bước tính toán hơn (đòi hỏi một mạng neural sâu hơn). Nhưng đây đều là các định nghĩa không chính thống.
```

Comment:
- Chính thống --> chính thức? 
- Thêm chữ "thường/đôi khi" tương ứng với "sometime" trong cụm 
> Chúng ta nói một tác vụ là "dễ" nếu có có thể được thực hiện với số lượng bước tính toán ít hơn (ứng với mạng neural nông)

- nếu có có thể được thực hiện với ~~số lượng~~ **_ít_** bước tính toán ~~ít ~~ hơn (ứng với mạng neural nông)